runtime/util_core/j9shchelp.c

```diff
@@ -258,14 +246,13 @@ getStringForShcModlevel(J9PortLibrary* portlib, uint32_t modlevel, char* buffer)
 	case J9SH_MODLEVEL_JAVA9 :
 		strcpy(buffer, "Java9");
 		break;
-	case J9SH_MODLEVEL_JAVA10 :
-		strcpy(buffer, "Java10");
-		break;
-	case J9SH_MODLEVEL_JAVA11 :
-		strcpy(buffer, "Java11");
-		break;
 	default :
-		strcpy(buffer, "Unknown");
+		if (modlevel > 9) {
```

Comment:
Even though its the same thing, I think its a little clearer to use `>= 10` since from 10 its different.

Reply:
Fixed